Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -962,90 +1009,26 @@ protected IEnumerator InputDownTimer(float time)
         /// <param name="eventData"></param>
         public void OnSpeechKeywordRecognized(SpeechEventData eventData)
         {
-            if (Enabled && ShouldListen(eventData.MixedRealityInputAction))
+            /* Internal Update (fe07a7aa Add support for localization speech commands) */
```

Comment:
This should be removed. It looks like it may be worth doing a scrub for the word "Internal"